docker/assets/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine
+
+# install the required dependencies to get assets
+RUN apk add curl
+RUN mkdir /assets
+RUN curl -L https://github.com/wowgaming/client-data/releases/download/v7/data.zip > "/assets/data.zip" \
```

Comment:
@Kapel would be better to use the `./acore.sh client-data` to download that data instead of this link directly

Reply:
@Yehonal this would require actually making a copy of acore.sh and main.sh (as acore.sh is just a wrapper) into the container. Also would have to copy apps dir inside. Just to call this single line, lol. If I add a variable for the v7 we can use it as a tag for tagging this image too.

Reply:
What about bind instead of copy, then unbind when everything done?

We have tons of scripts and it's easy to have breaking things if we don't remeber to change them everywhere

However, if you don't sugfest ir then you could create a variable inside config.sh with that link so that we can use it for both. Then you just need to copy the config.sh eventually.

Let me kniw

Reply:
Tuning in here, if I may: it might also be a good idea to join the various `RUN` steps into a single one (i.e. `RUN apk --no-cache add curl && mkdir -p /assets && ./acore.sh client-data`) in order to produce a single layer for the whole operation in the final Docker image (and adding `--no-cache` to `apk` to avoid bloating the Docker image with the package repo cache).

Reply:
@Yehonal looking at the function used in `./acore.sh client-data` it looks like it assumes the bin path which ends up being `./env/dist/bin/` I see the url is hard coded in the function as well. 

I am wondering if it might be a little easier to add it to the .env.dist and pass the url to the Dockerfile this way. similar to how we currently handle the "DB_ROOT_PASSWORD" for the database container. I think we do it this way we would not have to use copy or a bind mount. 

Then just update the client-data function to load in the url from the env.dist file. What do you think?